neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
A co by se dělalo v tom `self._init_attention()`? Teď se v tom konstruktoru jenom kontrolují ty `hasattr`. Ty když se vyhodí pryč, tak se `super()` může zavolat úplně na začátku konstruiktoru enkodéru. Důvod proč jsem to udělal takhle je, aby to spadlo už při inicializaci enkodéru, pokud by tam nevznikly správné tenzory a ne až při volání té metody `get_attention_object`, protože to se děje až v dekodéru a to by mohlo být matoucí potom při ladění konfiguráků.